config/initializers/session_store.rb

```diff
@@ -2,4 +2,4 @@
 
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: "_kudo_o_matic_session"
+# Rails.application.config.session_store :cookie_store, key: "_kudo_o_matic_session"
```

Comment:
Did you do this on purpose? I can imagine this application doesn't need sessions anymore, but in that case just remove the file instead of leaving commented code behind :-)